deploy/registration-service.yaml

```diff
@@ -18,26 +18,34 @@ objects:
       name: registration-service
       namespace: ${NAMESPACE}
     rules:
-    - apiGroups:
-      - toolchain.dev.openshift.com
-      resources:
-      - usersignups
-      verbs:
-      - create
-      - get
-    - apiGroups:
-      - toolchain.dev.openshift.com
-      resources:
-      - masteruserrecords
-      verbs:
-      - get
-    - apiGroups:
-      - toolchain.dev.openshift.com
-      resources:
-      - bannedusers
-      verbs:
-      - get
-      - list
+      - apiGroups:
+          - toolchain.dev.openshift.com
+        resources:
+          - usersignups
+        verbs:
+          - create
+          - get
+      - apiGroups:
+          - toolchain.dev.openshift.com
+        resources:
+          - masteruserrecords
+        verbs:
+          - get
+      - apiGroups:
+          - toolchain.dev.openshift.com
+        resources:
+          - bannedusers
+        verbs:
+          - get
+          - list
+      - apiGroups:
+          - ""
+        resources:
+          - configmaps
+          - secrets
+        verbs:
+          - get
+          - list
```

Comment:
thanks for the reminder @MatousJobanek i had mentioned it in the PR description but it's done here in case you missed it: https://github.com/codeready-toolchain/host-operator/pull/263/files